VL53L0X.h

```diff
@@ -132,6 +137,8 @@ class VL53L0X
     bool timeoutOccurred(void);
 
   private:
+	TwoWire *wire;
```

Comment:
The indentation of this is off, and it should be moved to be by the other private members below.